app/helpers.php

```diff
@@ -414,6 +414,31 @@ function get_valid_locale($requestedLocale)
     }
 }
 
+function hsl_to_hex($h, $s, $l)
+{
+    $s /= 100;
+    $l /= 100;
```

Comment:
as mentioned before, passing percentage value as is without the unit (`100%` as `100` instead of `1`) is weird. It's not how it works in the css either